metabase/metabase
# Fix whitelabeled application name replaces Metabase in admin settings

Closes https://github.com/metabase/metabase/issues/34920
[Slack convo](https://metaboat.slack.com/archives/C505ZNNH4/p1700480382374239)

### Description

We had [a discussion](https://metaboat.slack.com/archives/C01MS7DQKR6/p1700486165468699) that we'll only replace `Metabase` with user's application name (whitelabel) only outside the Metabase admin.

There are some places where we don't replace the application name:
1. Metabase admin
2. Setup flow: because the user who set up Metabase would already know they're using Metabase.
3. Powered by Metabase in embedding
4. ~[Database prompt banner:](https://www.notion.so/metabase/Really-encourage-admins-to-add-a-DB-9c8c3a7d08ca45cb8672a154220f443e) This should only appear when someone first set up a Metabase instance. They would know they're using Metabase.~ Actually @npretto has confirmed [we're not showing this when white labeling](https://github.com/metabase/metabase/blob/e09323b6d61b53533c7b0403ea5de6d71f02ffaa/frontend/src/metabase/nav/components/WhatsNewNotification/utils.ts#L23-L25) 🙏 .
5. [New Metabase version notification:](https://www.notion.so/metabase/Provide-a-heads-up-when-Metabase-is-upgraded-with-what-s-new-44a28b55624f48e28dc3f616ce7cda0f) because it'd be weird to try to tell your application has a new version when it will link to Metabase website anyway.
6. Pulse: because it's deprecated and there's no way to access the UI unless you enter it in the URL directly.

### How to verify
1. Run Enterprise version and set up the application
1. Check all places in the app and the string `Metabase` should be replaced with the user application name

### Demo

#### EE with token
![Screenshot 2023-11-22 at 9 16 58 PM](https://github.com/metabase/metabase/assets/1937582/e6bf8043-7384-4970-b21c-716ab4968e48)

### Checklist
